Lib/test/test_veronapy_writebarrier.py

```diff
@@ -76,3 +76,163 @@ def test_cell_replace_same_bridge(self):
         # since this writes replaces the previous owning reference
         c.cell_contents = child
 
+# It's not possible to test the existance of all bytecodes. The dis only
```

Comment:
dis?

Reply:
dis is a builtin module in c python to look at the bytecode.